models/points.go

```diff
@@ -1633,6 +1655,19 @@ func NewTags(m map[string]string) Tags {
 	return a
 }
 
+// Clone returns a shallow copy of Tags.
+//
+// Tags associated with a Point created by ParsePointsWithPrecision will hold references to the byte slice that was parsed.
+// Use Clone to create Tags with new byte slices that do not refer to the argument to ParsePointsWithPrecision.
+func (a Tags) Clone() Tags {
```

Comment:
Just a suggestion: `a` is not at risk of holding onto anything, if we remove all the references it contains. Therefore I don't think we need to allocate a new `Tags` value here. We can just do `a[i] = a[i].Clone()` and it will probably make this method significantly faster. 

Whether or not it's still appropriate to call this `Clone` with that implementation I'm not sure about. 😄